src/components/TodoWrapper.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+import { Todo } from "./Todo";
+import TodoForm from "./TodoForm";
+import { EditTodoForm } from "./EditForm";
+import type { TodoItem } from "../types/todo.types";
+
+export default function TodoWrapper() {
+  const [todos, setTodos] = useState<TodoItem[]>([]);
+  const [searchTerm, setSearchTerm] = useState<string>("");
+  const [filter, setFilter] = useState<"all" | "completed" | "incompleted">("all");
```

Comment:
Can we extract this union types.

Like: `type Status = "all" | "completed" | "incomplete"`

```suggestion
  const [filter, setFilter] = useState<"all" | "completed" | "incompleted">("all");
```

Reply:
fixed https://github.com/Sakshi-Simform/todo-list/pull/2/commits/01145ab40ab8df24db527b4b1ffee8427eff438c